influxql/decoder.go

```diff
@@ -0,0 +1,188 @@
+package influxql
```

Comment:
I don't think we want something so specifically `influxql`  in `flux`.   This code is probably a better fit in  functions/inputs  as a source decoder.  It would be OK to add a sub-package (even an internal on?) there to keep this file intact, and then write the source implementation using this as a helper package.

Reply:
also:  if you really want to implement this as a 2.0 style decoder, we probably need to keep this on `Platform` not `flux`